lib/src/main/java/com/auth0/android/lock/utils/json/Strategy.java

```diff
@@ -71,4 +73,15 @@ public boolean isActiveFlowEnabled() {
                 || Strategies.ADFS.getName().equals(name)
                 || Strategies.Waad.getName().equals(name);
     }
+
+    /**
+     * Returns the name of the first connection found in this strategy. When no connections available,
+     * it will default to the strategy name.
+     *
+     * @return the first connection found or the strategy name if no connections are available.
+     */
+    @NonNull
+    public String getDefaultConnectionName() {
+        return !connections.isEmpty() ? connections.get(0).getName() : this.name;
```

Comment:
This is not quite right for all strategies since returning the strategy name only applies to social ones. We should return `null` or avoid using a strategy without connections